sql/mysqld.cc

```diff
@@ -1998,6 +1998,7 @@ void clean_up(bool print_message)
   delete rpl_filter;
   end_ssl();
   vio_end();
+  vio_proxy_cleanup();
```

Comment:
Shouldn't this be called from `vio_end` instead, where all other vio related cleanups are executed? Its doc comment also states that it is the method which cleans up the memory allocated by vio.